proto/lbm/foundation/v1/tx.proto

```diff
@@ -44,36 +48,50 @@ service Msg {
 
   // Grant grants the provided authorization to the grantee with authority of
   // the foundation. If there is already a grant for the given
-  // (granter, grantee, Authorization) tuple, then the grant will be overwritten.
+  // (grantee, Authorization) tuple, then the grant will be overwritten.
   rpc Grant(MsgGrant) returns (MsgGrantResponse);
 
-  // Revoke revokes any authorization corresponding to the provided method name on the
-  // granter that has been granted to the grantee.
+  // Revoke revokes any authorization corresponding to the provided method name
+  // that has been granted to the grantee.
   rpc Revoke(MsgRevoke) returns (MsgRevokeResponse);
 
   // GovMint defines a gov mint coins to the treasury.
   rpc GovMint(MsgGovMint) returns (MsgGovMintResponse);
 }
 
-// MsgFundTreasury represents a message to fund the treasury.
+// MsgUpdateParams is the Msg/UpdateParams request type.
+message MsgUpdateParams {
+  // authority is the address of the operator account.
+  string authority = 1;
```

Comment:
It resembles the UpdateParams of cosmos-sdk, but in lbm-sdk, wouldn't the operator be better than the authority for consistency?

Reply:
I will deal with it on another PR. Refer to #741.